Cargo.lock

```diff
@@ -1379,6 +1400,50 @@ dependencies = [
  "zeroize",
 ]
 
+[[package]]
+name = "cxx"
```

Comment:
Hm, wat? 

I *guess* the code is unused on platforms we care about, but maybe we can avoid upgrading chrono to not pull these extra deps? 

I *think* chrono was also essentially abandoned in favor of time some time ago, but looks like it's being developed again? :weary: ideally, we'd just stick to std::SystemTime

Reply:
I don’t see how I convert date time into SystemTime.

Reply:
You can't: SystemTime is a unix timestamp, to convert that to/from human-readable time you need timezone information, and that's OS dependent and available only at runtime (because we don't statically know when leap seconds happen). 

If we didn't already depend on timezones (=chrono), I'd petition for either specifying this as a  `const` unix timestamp, with a comment containing a `date` invocation to convert human-readable text to timestamp, or for implementing approximate date-to-timestamp conversion which doesn't account for leap seconds as a const-fn (it's not like we care about utmost pecision )


:thinking: reading more about this, it seems that this might actually be far simpler in principle? I think I am wrong that "naive" conversion won't work. Reading 

https://blog.reverberate.org/2020/05/12/optimizing-date-algorithms.html

it seems like it's possible to implement

```rust
fn date_to_system_time(
  year: u16, month: u8, day: u8, hour: u8, minute: u8, second: u8
) -> SystemTime {
  SystemTime::UNIX_EPOCH + Duration::from_seconds(FORTAN formular from the post)
}
```

I don't know whether that's packaged somewhere already, let me ask...

Reply:
https://users.rust-lang.org/t/no-std-way-to-convert-from-human-readable-utc-time-to-unix-timestamp-and-back/82982

Reply:
SystemTime doesn’t seem to count leap seconds into account thus as you’ve observed in principle conversion between UTC date-time and SystemTime can be done via const function.  (In past life I have in fact implemented this in C++).  In fact it’s not possible to convert future dates to timestamps which counts leap seconds since they are known with only half a year advance notice.

However, more important question is do we want to move away from chrono.  From what I’ve gathered, it got undeprecated when it gained a new maintainer so we probably want to stay with it.

Reply:
PS. https://howardhinnant.github.io/date_algorithms.html is a good resource for all kinds of algorithms.